Kitodo/src/main/java/org/kitodo/export/ExportDms.java

```diff
@@ -205,9 +205,17 @@ private boolean prepareAndDownloadSaveLocation(Process process, URI destinationD
     private boolean exportImagesAndMetsToDestinationUri(Process process, LegacyMetsModsDigitalDocumentHelper gdzfile,
             URI destination, URI userHome) throws IOException {
 
-        boolean downloadImages = downloadImages(process, userHome, destination);
-        if (!downloadImages) {
-            return false;
+        if (exportWithImages) {
+            try {
+                directoryDownload(process, destination);
+            } catch (IOException | InterruptedException | RuntimeException e) {
+                if (Objects.nonNull(exportDmsTask)) {
+                    exportDmsTask.setException(e);
```

Comment:
there is no error message here

Reply:
That's true. What kind of error message should there be?

Reply:
How can I decide? You are the one to know, what is happening here.

Reply:
Of course, I know what's going on here: If an exception has occurred, it will be set in the executing thread for it to appear in the UI. If a message is to be set, it must happen at the point where the exception is thrown. Here only an existing exception is forwarded to the user interface. Therefore, I do not know what I should add here.